### PR TITLE
feat: populate and style create modal

### PR DESCRIPTION
**Pull request by alexnm**
This PR started as a data feature, but I ended up doing some styling/reorganization of the components in the create sandbox modal.

Implemented
- [x] Quick start page (3 recent templates + 6 selected from official list) (Closes XTD-102)
- [x] Cloud templates (official templates filter by `isV2: true`, no templates yet in this category (Closes XTD-105)
- [x] Official templates (the list that used to be in quick start before) (Closes XTD-103)
- [x] Cloud beta tag added (ticket still in progress, as the tag needs to be added in various places
- [x] Facelift for template cards, sidebar and second screen (Closes XTD-94)

image

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6940/commits/c6ad4079cea2d42e06d360e57a2262e5db01fd86 is at https://pr6940.build.csb.dev/s/new.

File: packages/app/src/app/components/CreateNewSandbox/CreateSandbox/useTeamTemplates.ts [hunk 1]
```diff
@@ -18,17 +26,15 @@ function getTeamTemplates(data: ListPersonalTemplatesQuery, teamId: string) {
   return data.me.teams.find(team => team.id === teamId).templates;
```
Comment (olarclara) [inline packages/app/src/app/components/CreateNewSandbox/CreateSandbox/useTeamTemplates.ts:26]: For some reason, on `/s` the `find` method returns `undefined` leading the app to crash when trying to access the `templates` property. Tested using the codesandbox team in production.

Screen Shot 2022-09-28 at 19 33 04

**Review by olarclara — COMMENTED** [review]
@alexnm nice work getting the data in!

the title mentions "style create modal", will the following screen/state be handled in a separate PR?

localhost_3000_dashboard

**Comment by alexnm** [comment]
> @alexnm nice work getting the data in!
> 
> the title mentions "style create modal", will the following screen/state be handled in a separate PR?
> 
> localhost_3000_dashboard

will work on it now and then merge the branch. I wanted to wrap up the work for the seamless branch switch, but now I still have time to work on it today

File: packages/app/src/app/components/CreateNewSandbox/CreateSandbox/useTeamTemplates.ts (hunk 1 above)
Comment (alexnm) [reply packages/app/src/app/components/CreateNewSandbox/CreateSandbox/useTeamTemplates.ts:26]: weird, what teams do you have there? maybe a data issue? i'm adding a check if the team is not found to return []

**Review by olarclara — APPROVED** [review]
🎨

**Pull request merged**